src/eduid_webapp/security/views/webauthn.py

```diff
@@ -123,13 +123,25 @@ def registration_complete(user, credential_id, attestation_object, client_data,
 @require_user
 def remove(user, credential_key):
     security_user = SecurityUser.from_user(user, current_app.private_userdb)
-    token_to_remove = security_user.credentials.filter(Webauthn).find(credential_key)
+    tokens = security_user.credentials.filter(U2F).to_list()
```

Comment:
filter(FidoCredential) will work instead of doing two lookups and a concatenation.